run-elrond-tests.py

```diff
@@ -24,6 +24,11 @@ def KWasmString(value):
 def KInt(value : int):
     return KToken(str(value), 'Int')
 
+def KBytes(value: bytes):
+    # Change from python bytes repr to bytes repr in K.
+    byte_repr = '{}'.format(value).replace("'", '"')
+    return KToken(byte_repr, 'Bytes')
```

Comment:
I didn't know this was possible. If this works (and the PR passes), can you upstream it into the KWasm bytes parser? https://github.com/kframework/wasm-semantics/blob/master/binary-parser/kwasm_ast.py#L77